monkey/monkey_island/cc/services/agent_binary_service/agent_binary_service.py

```diff
@@ -12,8 +14,19 @@ class AgentBinaryService(IAgentBinaryService):
     def __init__(self, agent_binary_repository: IAgentBinaryRepository):
         self._agent_binary_repository = agent_binary_repository
 
+        self._os_masque: Dict[OperatingSystem, Optional[bytes]] = {
+            OperatingSystem.LINUX: None,
+            OperatingSystem.WINDOWS: None,
+        }
+
     def get_linux_binary(self) -> BinaryIO:
         return self._agent_binary_repository.get_linux_binary()
 
     def get_windows_binary(self) -> BinaryIO:
         return self._agent_binary_repository.get_windows_binary()
+
+    def get_masque(self, os: OperatingSystem) -> Optional[bytes]:
+        return self._os_masque.get(os, None)
+
+    def set_masque(self, masque: Optional[bytes], os: OperatingSystem):
+        self._os_masque[os] = masque
```

Comment:
1. Reverse parameter order.
2. Avoid potential naming collision with the `os` library.